### ISSUE TITLE
does pytorch1.0 support Synchronized BatchNorm?

### ISSUE THREAD

**Issue by mlinxiang** [facebookresearch/maskrcnn-benchmark#34]
## 🚀 Feature
does pytorch1.0 support Synchronized BatchNorm, and in this code the FrozenBatchNorm has the same function as Synchronized BN?

why use FrozenBatchNorm, as same thirdparty faster rcnn implement's backbone resnet do'nt has this feature

**Comment by soumith** [comment]
Added now in PyTorch master via https://github.com/pytorch/pytorch/pull/14267
For documentation, see: https://pytorch.org/docs/master/nn.html#torch.nn.SyncBatchNorm
